### DISCUSSION TITLE
ChatItem - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#201] [Q&A]
# ChatItem - Lobe UI

ChatItem is a React component that represents a single item in a chat conversation. It displays the user's avatar, name, and message. It can also display a loading indicator if the message is still being sent.

https://ui.lobehub.com/components/chat-item

**Comment by Lnncoco** [comment]
- editing = true 时出来的那个编辑组件，如何修改取消、确定按钮文字？同时翻了文档，试了几个值，也没找到怎么分别响应取消、确定的按钮事件，只找到个 onEditingChange 接收改变的事件。MessageInput 有看到事件属性，但在这里传不进去。
- `role` 值非常多，单纯的改了值试了试，没有出现预期的样式效果，比如希望出现在中间变成系统提示效果，没找到实现方式，希望提供一些更详细的示例。
- 属性值真的太多了，说明太简单，很难查找，一边翻源码一遍看。按ctrl+f搜索会变成全局搜索，没办法在页内搜索。
